PyDMXControl/profiles/defaults/_Fixture.py

```diff
@@ -129,6 +136,12 @@ def locate(self):
         self.color([255, 255, 255, 255, 255])
         self.dim(255)
 
+    def remove_local_callbacks(self):
```

Comment:
```suggestion
    def clear_callbacks(self):
```
Naming consistency with the effects method